imod/mf6/utilities/logging_decorators.py

```diff
@@ -0,0 +1,52 @@
+from imod.logging import ILogger, logger
```

Comment:
I would put this file in the logger module

Reply:
done